corehq/messaging/scheduling/static/scheduling/js/create_schedule.js

```diff
@@ -1,11 +1,11 @@
-hqDefine("scheduling/js/create_schedule.ko", [
+hqDefine("scheduling/js/create_schedule", [
     'jquery',
     'knockout',
     'hqwebapp/js/initial_page_data',
     'hqwebapp/js/select2_handler',
     'jquery-ui/ui/widgets/datepicker',
     'bootstrap-timepicker/js/bootstrap-timepicker',
-    'hqwebapp/js/ckeditor_knockout_bindings',
+    'hqwebapp/js/ckeditor_knockout_bindings',       // TODO: only include if feature flag is on
```

Comment:
Is this TODO expected to be done sometime soon?

Reply:
Removed this commit.